core/storage/user/gae_models.py

```diff
@@ -195,6 +195,25 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def export_data(user_id):
+        """(Takeout) Export CompletedActivitiesModel's user properties.
+        Args:
```

Comment:
Add line above Args; add line above Returns. In general please follow existing styles.

Ditto elsewhere.

Reply:
Ack

Reply:
Hi @ulloaluis -- I'm not sure what "Ack" means; usually it's used when the developer has read the comment, acknowledges its receipt, but doesn't do anything about it. That's not the case here, right?

In any case, for comments that you are acting on, could you please reply with "Done" (after you have submitted the fix)? Thanks.

Reply:
Oh, I thought it just meant "acknowledged" and had no other connotations. Yes, everywhere I said "ack" I mean "done."